recipes/pysirius/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "0.9" %}
+{% set name = "py-sirius-ms" %}
+{% set commit = "027828d27624b94f0d99885eee2ae45301a9b9fb" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/boecker-lab/sirius-client-openAPI/archive/{{ commit }}.zip
+  sha256: 404cd1d1a9eb660407128548db4da2cda0b37d2115db260402d82cad33ab45d4
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install ./client-api_python/generated -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+
+  run:
+    - python >=3.8
+    - certifi >=14.05.14
+    - six >=1.10
+    - python-dateutil >=2.5.3
+    - setuptools >=21.0.0
+    - urllib3 >=1.15.1
+    - sirius-ms
+
+test:
+  imports:
+    - PySirius
+  requires:
+    - pip
+  commands:
+    - pip check
+    - sh run_test.sh           # [not win]
+    - call run_test.bat        # [win]
```

Comment:
Having these scripts named `run_test.*` tells `conda-build` (counterintuitively) to ignore the `test.commands` section. Please rename to something else like `test_script.*` or something like that so `pip check does run.